neusomatic/python/train.py

```diff
@@ -219,7 +219,17 @@ def train_neusomatic(candidates_tsv, validation_candidates_tsv, out_dir, checkpo
         torch.set_num_threads(num_threads)
 
     data_transform = matrix_transform((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+
+    ensemble = False
+    with open(candidates_tsv[0]) as i_f:
+        for line in i_f:
+            x=line.strip().split()
+            if len(x)==97:
```

Comment:
@msahraeian should define package / global constants instead of hardcoding numbers. Would make it easier to update these values if you add more columns. Can also have a function to guess whether ensemble or not and move this code there.